src/webui/api/rsscontroller.cpp

```diff
@@ -91,7 +94,28 @@ void RSSController::itemsAction()
     setResult(jsonVal.toObject());
 }
 
-void RSSController::refreshItemAction()
+void RSSController::markAsReadAction() const
+{
+    requireParams({"itemPath"});
+
+    const QString itemPath {params()["itemPath"]};
+    const QString articleId {params()["articleId"]};
+
+    RSS::Item *item = RSS::Session::instance()->itemByPath(itemPath);
+
+    if (!item) return;
+
+    if(!articleId.isEmpty()) {
```

Comment:
Broken coding style.
You should consider qBittorrent Coding Guidelines.

Reply:
There are two cases:
1. "articleId" is missing in parameters
2. "articleId" is set as empty string (i.e. `articleId=`)

IMO, the second case is invalid request. I would handle it as error and reject.

Reply:
Im checking now if parameters contain articleId instead of checking if empty.
In the second case no article with will be found and thus function will return. Later trow APIError could be implemented.